### DISCUSSION TITLE
Issuesとかで数式を書く方法

### DISCUSSION THREAD

**Discussion by homoscribens** [homoscribens/Paper-Introduction-JAIST2022#10] [Q&A] [answered]
画像貼り付けるしかないんですかね

**Comment by kan25d5** [comment]
調べたらこういうのがあった．
結局画像使うことになるけど，altに数式を読める形で記述してくれるのはありがたい．

githubやnoteでもTeXの数式を書くぜ - 学習する天然ニューラルネット
https://aotamasaki.hatenablog.com/entry/2020/08/09/github%E3%82%84note%E3%81%A7%E3%82%82TeX%E3%81%AE%E6%95%B0%E5%BC%8F%E3%82%92%E6%9B%B8%E3%81%8F%E3%81%9C

**Reply by homoscribens** [reply]
ありがと
良さげやけど，ページの背景色によって見えなくなるのが惜しいなぁ．．．

**Comment by homoscribens** [comment, marked as answer]
https://github.blog/2022-05-19-math-support-in-markdown/

Tex数式サポートされました
解決！
